backend/src/main/resources/config/liquibase/changelog/initial_changelog.xml

```diff
@@ -18,7 +18,7 @@
             <column name="size" type="BIGINT">
                 <constraints nullable="false"/>
             </column>
-            <column name="feature_types" type="VARCHAR"/>
+            <column name="column_meanings" type="VARCHAR"/>
```

Comment:
you shouldn't be modifying existing database patches as they are already applied on users' servers. We have to generate a new patch instead. you can do it with `../gradlew liquibaseCreateDiff` from `backend` directory